test/SwooleHttpServerFactoryTest.php

```diff
@@ -48,7 +48,7 @@ public function testInvokeWithoutConfig()
     public function testInvokeWithConfig()
     {
         $config = [
-            'swoole' => [
+            'swoole_http_server' => [
```

Comment:
In one of my last PRs, I updated the code to use namespaced classes everywhere, instead of the `underscore_variants`. As such, I wonder if we should use `Swoole\Http\Server` here, for consistency?

Reply:
@weierophinney as we discussed in chat, we'll change it using the following syntax (to be consistent with [zend-expressive-hal](https://github.com/zendframework/zend-expressive-hal/blob/master/src/ConfigProvider.php#L32)):
```
'zend-expressive-swoole' => [
    'swoole-http-server' => [
        // ...
    ]
]
```